posthog/settings.py

```diff
@@ -731,3 +732,13 @@ def show_toolbar(request):
 
 # keep in sync with plugin-server
 EVENTS_DEAD_LETTER_QUEUE_STATSD_METRIC = "events_added_to_dead_letter_queue"
+
+SERVICE_VERSION_REQUIREMENTS = [
+    ServiceVersionRequirement(service="postgresql", supported_version=">=11.0.0,<15.0.0",),
+    ServiceVersionRequirement(service="redis", supported_version=">=5.0.0,<7.0.0",),
+]
+
+if PRIMARY_DB == AnalyticsDBMS.CLICKHOUSE:
+    SERVICE_VERSION_REQUIREMENTS = SERVICE_VERSION_REQUIREMENTS + [
+        ServiceVersionRequirement(service="clickhouse", supported_version=">=21.6.0,<21.7.0"),
```

Comment:
This is a tricky one, I think cloud is where we run the oldest, Helm runs on `yandex/clickhouse-server:21.6.5`...

Reply:
Should be fine to support an entire minor range?

Reply:
Sure, let's just make sure clouds runs on >=21.6 or this will fail

Reply:
Yup, Cloud is: `21.6.5.37`